core/templates/dev/head/pages/exploration_editor/ExplorationEditor.js

```diff
@@ -182,6 +182,9 @@ oppia.controller('ExplorationEditor', [
           EditabilityService.markTranslatable();
         }
 
+        StateEditorService.setExplorationIsWhitelisted(
```

Comment:
updateExplorationWhitelistedStatus

(otherwise, sounds like you're setting it to true in all cases)

Reply:
Done.